proto/src/determined/api/v1/api.proto

```diff
@@ -500,16 +500,86 @@ service Determined {
       tags: [ "Trials", "Experiments" ]
     };
   }
+
   // Return downsampled metrics from multiple trials to compare them
   // side-by-side.
   rpc CompareTrials(CompareTrialsRequest) returns (CompareTrialsResponse) {
     option (google.api.http) = {
       get: "/api/v1/trials/compare"
     };
     option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
-      tags: [ "Trials", "Experiments" ]
+      tags: "Experiments"
+    };
+  }
+
+  // Return trials matching a set of TrialFilters
+  rpc QueryTrials(QueryTrialsRequest) returns (QueryTrialsResponse) {
+    option (google.api.http) = {
+      post: "/api/v1/trials/query"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: [ "Trial Comparison" ]
+    };
+  }
+
+  // Update tags for trials by the trial ID
+  rpc PatchTrials(PatchTrialsRequest) returns (PatchTrialsResponse) {
+    option (google.api.http) = {
+      patch: "/api/v1/trials"
+      body: "*"
+    };
+    option (grpc.gateway.protoc_gen_swagger.options.openapiv2_operation) = {
+      tags: [ "Trial Comparison" ]
+    };
+  }
```

Comment:
> Switch existing /api/v1/trials/compare to /api/v1/trial-comparison/compare as a tech debt item.

"tech debt item" = separate ticket?

Reply:
> Looking at this from an API user perspective, if I discovered an API called "PatchTrials" I'd not be expecting it to do what it does in this PR. So I'd suggest to change to a different, more specific name, e.g. UpdateTrialTags

The thought was to potentially allow for different updates in the future, but this makes sense for now.

Reply:
>"tech debt item" = separate ticket?

If you could change it in this PR it'd be stellar. Otherwise yes let's please have a tracking ticket for that.

Reply:
it would require frontend changes as well since that API is already being used by unrelated work. 

I would rather keep this scoped to just backend changes if that is okay.

Reply:
sounds good.